manual/src/main/asciidoc/user-guide/collectors.adoc

```diff
@@ -472,11 +472,28 @@ This feature installs a default `etc/org.apache.karaf.decanter.collector.socket.
 
 # Protocol tcp(default) or udp
 #protocol=tcp
+
+# Dataformat json(default) : 
+# - The dataformat has to match the unmarshaller property in the component annotation
+# - Example : property = Marshaller.SERVICE_KEY_DATAFORMAT + "=json"
+unmarshaller.target=(dataformat=json)
 ----
 
 * the `port` property contains the port number where the network socket collector is listening
 * the `workers` property contains the number of worker thread the socket collector is using for connection
 * the `protocol` property contains the protocol used by the collector for transferring data with the client
+* the `dataformat` property contains the unmarshaller used by the collector to transform the data
+sended by the client. The custom unmarshaller must be declared as a component and set the service property correctly.
+
+In order to create a custom unmarshaller, we create the following `MyCustomUnmarshaller`  class implementing `org.apache.karaf.decanter.api.marshaller.Unmarshaller` interface:
```

Comment:
I don't think it's useful to document marshaller/unmarshaller there. I will add a specific section about it, but not in the scope of this PR.